traitsui/qt4/tabular_editor.py

```diff
@@ -913,7 +926,8 @@ def columnResized(self, index, old, new):
             if self._user_widths is None:
                 self._user_widths = [None] * len(self._editor.adapter.columns)
             self._user_widths[index] = new
-            if not self._editor.factory.auto_resize:
+            if (self._editor.factory is not None
```

Comment:
Actually I can fix this properly than working around it.

Reply:
Decided not to eventually so this PR is more trivial and incur lower risk for it to be backported to the bug fix release.